app/eventyay/config/settings.py

```diff
@@ -45,8 +45,22 @@ def instance_name(request):
 debug_fallback = 'runserver' in sys.argv
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
-DATA_DIR = config.get('eventyay', 'datadir', fallback=os.environ.get('DATA_DIR', 'data'))
+DATA_DIR = os.environ.get(
+    'DATA_DIR',
+    config.get('eventyay', 'datadir', fallback='data')
+)
+LOG_DIR = os.path.join(DATA_DIR, 'logs')
+MEDIA_ROOT = os.path.join(DATA_DIR, 'media')
+PROFILE_DIR = os.path.join(DATA_DIR, 'profiles')
 BASE_PATH = ''
+STATIC_URL = config.get('urls', 'static', fallback=BASE_PATH + '/static/')
```

Comment:
Please don't make `STATIC_URL`, `MEDIA_URL` configurable.
The more configuration, the more pain for maintenance.